api/DataAccess/DataAccessObjects/UserDataAccessObject.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Data.Context;
+using Data.Entities;
+using DataAccess.ExtensionMethods;
+using DataAccess.Interfaces;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataAccess.DataAccessObjects
+{
+    public class UserDataAccessObject : IUserDataAccessObject
+    {
+        protected readonly UnoOnBoardingContext _context;
+
+        public UserDataAccessObject(UnoOnBoardingContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<List<User>> FilterUsers(string search,int sort)
+        {
+            search = search.RemoveDiacritics();
+            if (sort == 0)
+            {
+                List<User> result = await _context.Set<User>()
+                .Where(x => (x.Name!.ToLower().Contains(search.ToLower())) ||
```

Comment:
Tens que remover no Name tbm, porque na bd pode estar o nome Antônio

Reply:
nao consigo aplicar estes metodos a queries à BD, penso que a unica soluçao seja mesmo nas definiçoes da tabela da BD

Reply:
Mas não podes fazer isto:  x.Name!.ToLower().RemoveDiacritics().Contains(search.ToLower())????

Reply:
não, pq nao dá para converter isso numa query sql
https://stackoverflow.com/questions/7418624/linq-where-ignore-accentuation-and-case

Reply:
done